packages/vite/src/node/server/transformRequest.ts

```diff
@@ -187,7 +192,45 @@ async function loadAndTransform(
 
   // load
   const loadStart = debugLoad ? performance.now() : 0
-  const loadResult = await pluginContainer.load(id, { ssr })
+  let loadResult: LoadResult
+
+  const cacheLoadResult = await pluginContainer.serveLoadCacheRead({
```

Comment:
it looks like this happens in both dev and build in which case I don't think we should call it "serve" because it's a bit confusing with Vite's "serve" command (https://vitejs.dev/config/#conditional-config)

maybe we could call it something like `getFromLoadCache`